src/Microsoft.DotNet.ProjectModel/FileFormatException.cs

```diff
@@ -24,7 +24,7 @@ public sealed class FileFormatException : Exception
         
         public override string ToString()
         {            
-            return $"{Path}({Line},{Column}): Error: {base.ToString()}";
+            return string.Format("{0}({1},{2}): {3}", Path, Line, Column, base.Message);
```

Comment:
Keep throwing exception in string interpolated form from Console.Logger. I try to avoid the issue here first.